primitives/keyring/src/dilithium2.rs

```diff
@@ -213,6 +210,6 @@ mod tests {
 		let alice = Keyring::Alice.public();
```

Comment:
remove `#[ignore]` and `//TODO` if this test passes

Reply:
Done